assets/components/button/button.scss

```diff
@@ -83,26 +80,32 @@
 }
 
 .component-button svg {
-  width: $gu-h-spacing * 1.075;
-  margin: $gu-v-spacing * -1 $gu-h-spacing * -.5 $gu-v-spacing * -1 $gu-h-spacing * .5;
+  fill: currentColor;
```

Comment:
this might wreck havoc with most of our svgs because they aren't properly flattened! I'm down with making this change and fix the uglies later but want to make sure you are aware